src/helpers/UndoManager/useUndoManager.js

```diff
@@ -46,6 +46,7 @@ const useUndoManager = (
         canRedo: nextRedo,
       });
     });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
If possible, solve this so that the disable is not required

Reply:
Sure!... adding dispatch to the dependency list will do ig.